bundles/CoreBundle/src/Migrations/Version20221116115427.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use Pimcore\Model\User;
+use Pimcore\Model\User\Listing;
+
+
+final class Version20221116115427 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add "object bricks" permission';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql("INSERT INTO `users_permission_definitions` (`key`, `category`) VALUES ('objectbricks', '')");
+
+        $this->addSql('UPDATE `users` SET `permissions`=CONCAT(`permissions`, \',objectbricks\') WHERE `permissions` REGEXP \'(?:^|,)classes(?:^|,)\'');
```

Comment:
Unfortunately this regex does not cover all cases.
Since the user could only have the classes permissions, the permissions text would look like this
`classes` without any commas.
It could also be the first permission or the last permission:
`classes,translations` first
`assets,classes` last

```suggestion
        $this->addSql('UPDATE `users` SET `permissions`=CONCAT(`permissions`, \',objectbricks\') WHERE `permissions` REGEXP \'(?:|,)classes(?:|,)\'');
```

Reply:
The (?:^|,) at the end should have been (?:$|,) which means "end of string or comma". Have fixed it.